department-of-veterans-affairs/va.gov-cms
# VACMS-14955: Hide ief_reference_save button for CLP

## Description

Relates to #14955.

## QA steps

As a user with access to create Campaign Landing Pages
1. Add a new Campaign Landing Page (/node/add/campaign_landing_page)
2. Scroll down to 'Downloadable Resources' and open the section.
3. Click to 'Enable this page segment'
4. Under the 'Resources' section click the 'Add existing Downloadable resource'
- [x] Verify that there are now only 2 buttons: 'Select Downloadable resource' and 'Cancel'
5. Click to 'Select Downloadable resource' and choose a file in the modal that pops up.
- [x] Verify that you can select a resource and it then shows up in the table.
6. Click to 'Add existing Downloadable resource' again.
7. Click to 'Select Downloadable resource' and choose another file.
- [x] Verify that a second file is added to the table.

### Definition of Done


### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing announcement

Is an announcement needed to let editors know of this change?
